diff --git i/frontend/src/components/auth/utils/registration-alert.jsx w/frontend/src/components/auth/utils/registration-alert.jsx
index 3c4b91c..50168a4 100644
--- i/frontend/src/components/auth/utils/registration-alert.jsx
+++ w/frontend/src/components/auth/utils/registration-alert.jsx
@@ -113,7 +113,7 @@ export function ConfigurationModalAlert(props) {
           <section className="mt-4">
             <p>
               If you are an admin please check the{" "}
-              <a href={GREEDYBEAR_DOCS_URL} target="_blank" rel="noreferrer">
+              <a href={GREEDYBEAR_DOCS_URL} target="_blank" rel="noopener noreferrer">
                 documentation
               </a>{" "}
               and correctly configure all the required variables.
diff --git i/frontend/src/components/feeds/Feeds.jsx w/frontend/src/components/feeds/Feeds.jsx
index f81c276..149a31b 100644
--- i/frontend/src/components/feeds/Feeds.jsx
+++ w/frontend/src/components/feeds/Feeds.jsx
@@ -138,6 +138,7 @@ export default function Feeds() {
           outline
           href={FEEDS_LICENSE}
           target="_blank"
+          rel="noopener noreferrer"
         >
           <TbLicense />
           &nbsp;Feeds license
@@ -243,6 +244,7 @@ export default function Feeds() {
               outline
               href={url}
               target="_blank"
+              rel="noopener noreferrer"
             >
               <VscJson />
               &nbsp;Raw data
diff --git i/frontend/src/layouts/widget/UserMenu.jsx w/frontend/src/layouts/widget/UserMenu.jsx
index 6c7f486..1e6c12c 100644
--- i/frontend/src/layouts/widget/UserMenu.jsx
+++ w/frontend/src/layouts/widget/UserMenu.jsx
@@ -29,7 +29,7 @@ function UserMenu(props) {
         </DropdownItem>
         <DropdownItem divider />
         {/* Django Admin Interface */}
-        <DropdownNavLink to="/admin/" target="_blank">
+        <DropdownNavLink to="/admin/" target="_blank" rel="noopener noreferrer">
           <IoMdSettings className="me-2" /> Django Admin Interface
         </DropdownNavLink>
         {/* API Access/Sessions */}
